locale/pl/LC_MESSAGES/disco.po

```diff
@@ -15,91 +15,93 @@ msgstr ""
 
 #: src/disco/components/Addon.js:112
 msgid "Close"
-msgstr ""
+msgstr "Zamknij"
 
 #: src/disco/components/Addon.js:141
 msgid "Hover to preview or click to install %(name)s"
 msgstr ""
+"Najechanie wyświetli podgląd, kliknięcie zainstaluje dodatek „%(name)s”"
```

Comment:
How this string is used? Wouldn't it be better to translate it more like "Najedź, aby wyświetlić podgląd"?